### DISCUSSION TITLE
Preact support

### DISCUSSION THREAD

**Discussion by LoganDark** [JohannesKlauss/react-hotkeys-hook#1107] [Ideas]
I'm working on a Preact app, and this library looks useful and feature-complete, but it uses React types directly and I see no mentions of Preact anywhere in the documentation or issue tracker. Would it be possible to add Preact support to the library? AFAIK, it would mostly consist of using Preact hooks instead of React's, and ingesting native browser key events rather than React's synthetic events.

**Comment by JohannesKlauss** [comment]
Sorry, I barely have time to invest development in the library as is, so I can't do any Preact support. I am happy to take contributions, though. It is open source after all.
